dom.bs

```diff
@@ -8715,10 +8706,9 @@ these steps:
  <!-- !((1 << (node.nodeType - 1)) & whatToShow) -->
  <li>If <a for=traversal>filter</a> is null,
  return {{NodeFilter/FILTER_ACCEPT}}.
- <li>Let <var>result</var> be the return value of calling
- <a for=traversal>filter</a>'s
- {{NodeFilter/acceptNode()}} with <var>node</var> as
- argument. Rethrow any exceptions.
+ <li>Let <var>result</var> be the return value of <a>call a user object's operation</a> with
+ <a for=traversal>filter</a>, "<code>acceptNode</code>", and a list of arguments consisting of
```

Comment:
Yeah, but it can be a string that links nicely... Similar to how in https://dom.spec.whatwg.org/#dfnReturnLink-3 we use `<{html}>` as a string (local name) and it seems OK.